include/sys/arc_impl.h

```diff
@@ -206,7 +206,6 @@ typedef struct l2arc_log_blkptr {
  */
 typedef struct l2arc_dev_hdr_phys {
 	uint64_t	dh_magic;	/* L2ARC_DEV_HDR_MAGIC */
-	uint64_t	dh_version;	/* Persistent L2ARC version */
```

Comment:
To clarify a little bit, I'm by no means against keeping this on-disk version information.  In fact, I think we should keep it in case we find ourselves wanting to store additional state in the future.  All I was suggesting is that we need not expose this version number to user space via sysfs.  This is a low level detail the user space utilities don't need to be aware of.